railties/lib/rails/commands/help/help_command.rb

```diff
@@ -8,8 +8,19 @@ class HelpCommand < Base # :nodoc:
       def help(*)
         say self.class.desc
 
-        Rails::Command.print_commands
+        other_commands = printing_commands_not_in_usage.sort_by(&:first)
```

Comment:
```suggestion
        other_commands = printing_commands_not_in_use.sort_by(&:first)
```
Great PR, but this English typo's been driving me nuts so let's fix it while we're at it. 

See: https://english.stackexchange.com/a/51651/65369

In summary: 
- "this is not proper usage" ✅
- "this command is not in use" ✅
- "this command is not in usage" 💥

Reply:
@olivierlacan Thanks for the review 😄 

This was confusing to me as well, but usage refers to the USAGE file:
https://github.com/rails/rails/blob/master/railties/lib/rails/commands/help/USAGE
There are a lot of places in the code that refer to the usage, for example:
https://github.com/rails/rails/blob/master/railties/lib/rails/command/base.rb#L29-L37

Maybe we can rename it to `printing_commands_not_in_usage_file`?

Reply:
Damn it! 😂

That’s a good idea.